cli/plasmo/templates/static/vue3/content-script-ui-mount.ts

```diff
@@ -0,0 +1,82 @@
+// @ts-nocheck
```

Comment:
These are more or less template code tho. Having a factory for them wouldn't work unless we throw them in somehow with the compiler (i.e, make a compiler xd...) 

They repeat each other now, but in the long run I doubt that'd be the case tbh. Some of these code actually might not work. Ppl who use it will tell about it doh ;p